central/telemetry/billingmetrics/data.go

```diff
@@ -0,0 +1,69 @@
+package billingmetrics
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	bmetrics "github.com/stackrox/rox/central/billingmetrics/store"
+	cluStore "github.com/stackrox/rox/central/cluster/datastore"
```

Comment:
cluStore -> clusterStore aligns more closely with our general syntax

Reply:
I'm pretty sure it's intended as a word play: cluster + store = "clustore", pronounced almost like cluster but not quite.